File: swagger/models/credit_card_info.py
from .base_model import BaseModel


class CreditCardInfo(BaseModel):
	"""
	INformation about an individual credit card

	Attributes:
	"""

	input_map = {
		'CreditCardNumber': 'credit_card_number',
		'ExpMonth': 'exp_month',
		'ExpYear': 'exp_year',
		'BillingName': 'billing_name',
		'BillingAddress': 'billing_address',
		'BillingCity': 'billing_city',
		'BillingState': 'billing_state',
		'BillingPostalCode': 'billing_postal_code',
		'SaveInfo': 'save_info',
		}

	output_map = {
		'credit_card_number': 'CreditCardNumber',
		'exp_month': 'ExpMonth',
		'exp_year': 'ExpYear',
		'billing_name': 'BillingName',
		'billing_address': 'BillingAddress',
		'billing_city': 'BillingCity',
		'billing_state': 'BillingState',
		'billing_postal_code': 'BillingPostalCode',
		'save_info': 'SaveInfo',
		}

File: swagger/Models/CreditCardInfo.php
<?php

namespace App\MindBody\Models;

/**
 * INformation about an individual credit card
 */
class CreditCardInfo extends BaseModel
{
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $creditCardNumber;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $expMonth;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $expYear;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $billingName;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $billingAddress;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $billingCity;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $billingState;
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $billingPostalCode;
	/**
	 * no description available
	 * 
	 * @var boolean
	 */
	public $saveInfo;

	protected function getInputMap()
	{
		return [
			'CreditCardNumber' => 'creditCardNumber',
			'ExpMonth' => 'expMonth',
			'ExpYear' => 'expYear',
			'BillingName' => 'billingName',
			'BillingAddress' => 'billingAddress',
			'BillingCity' => 'billingCity',
			'BillingState' => 'billingState',
			'BillingPostalCode' => 'billingPostalCode',
			'SaveInfo' => 'saveInfo',
			];
	}

	protected function getOutputMap()
	{
		return [
			'creditCardNumber' => 'CreditCardNumber',
			'expMonth' => 'ExpMonth',
			'expYear' => 'ExpYear',
			'billingName' => 'BillingName',
			'billingAddress' => 'BillingAddress',
			'billingCity' => 'BillingCity',
			'billingState' => 'BillingState',
			'billingPostalCode' => 'BillingPostalCode',
			'saveInfo' => 'SaveInfo',
			];
	}
}
